### PR TITLE
users/strelaysrv: Document new access control, update options

### PR DESCRIPTION
**Pull request by GermanCoding**
Documentation for PR https://github.com/syncthing/syncthing/pull/8561

Also took the opportunity to document missing flags `-pprof` and `-version`

### CODE REVIEW

File: users/strelaysrv.rst [hunk 1]
```diff
@@ -89,6 +89,10 @@ Options
     "https://relays.syncthing.net/endpoint"). Blank to disable announcement to
     a pool, thereby remaining a private relay.
 
+.. cmdoption:: -pprof
+
+    Enable the built in profiling on the status server
+
```
Comment (Riolku) [inline users/strelaysrv.rst:95]: Does this mean the `Go` profiler? Would this dump profile data? Or is this the `/status` endpoint - wasn't that enabled by default?

Maybe it's just me but I don't understand this sentence as written :)

Comment (GermanCoding) [reply users/strelaysrv.rst:95]: This was copy-pasted from the `strelaysrv -help` output. It's in the relaysrv source code here: https://github.com/syncthing/syncthing/blob/81a4b22d4342b187ca910a1e6f4ce0ff61b89569/cmd/strelaysrv/main.go#L102

What it does is it enables a new `/debug/pprof` endpoint you can request to get a Go profile (see the code here: https://github.com/syncthing/syncthing/blob/81a4b22d4342b187ca910a1e6f4ce0ff61b89569/cmd/strelaysrv/status.go#L24). I've no idea if this is documented anywhere (probably not), but it's similar to how it's handles in the main syncthing service (that also has a rest/debug/pprof endpoint you can enable).

Comment (Riolku) [reply users/strelaysrv.rst:95]: Mildly feel like this wording is weird, I might say "enable go's builtin profiler" or something, but if it's copy pasted from elsewhere I feel like it's fine to keep it.. probably.

**Comment by st-review** [comment]
:robot: beep boop

I'm going to close this pull request as it has been idle for more than 90 days.

This is not a rejection, merely a removal from the list of active pull requests that is periodically reviewed by humans. The pull request can be reopened when there is new activity, and merged once any remaining issues are resolved.

To permanently exempt this PR from bot nagging, add the `slow-pr` label.

**Review by calmh — APPROVED** [review]

**Pull request merged**